### DISCUSSION TITLE
[Question]: Conversation not found with Assistents

### DISCUSSION THREAD

**Discussion by samuelpatro** [danny-avila/LibreChat#2765] [Troubleshooting]
### What is your question?

Creating a assistant returns everything fine then selecting it and asking him first question it returns in request just "Conversation not found". After refreshing all assistant dissapears, but checking on openai website in API for assistents it exists. 

Can this be because of something is misconfigured?

### More Details

Latest version

Error message from log: 2024-05-17 10:11:01 error: [handleAbortError] AI response error; aborting request: Assistant not supported

### What is the main subject of your question?

Installation, Endpoints

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/3110002/f256e506-409d-4249-a742-5101bd26c6f1)
![image](https://github.com/danny-avila/LibreChat/assets/3110002/1b12434e-17ab-4b17-a2d7-e136faa6436f)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samuelpatro** [comment]
Looks like we solved it, includedIds we switched to excludedIds and it works. Magic :O

**Comment by danny-avila** [comment]
How were you selecting it? Because I can't reproduce once I create, then select it.

**Reply by samuelpatro** [reply]
Just clicked Select button, then texted to it something, after it responded error i refreshed and assistant dissapeared. Maybe something was misconfigured in yaml file, i findout after time there was error with spacings.

**Reply by danny-avila** [reply]
it might be an issue with the excluded/included filters. It makes sense that having them configured would lead to errors when creating them.
